posts/2020/03/Spyder-4.md

```diff
@@ -0,0 +1,31 @@
+<!--
+.. title: Spyder 4 is here!
+.. slug: spyder-4-is-here
+.. date: 2020-03-05 14:00:00 UTC-05:00
+.. author: Carlos Córdoba
+.. tags: Labs, Spyder
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+After more than three years in development, and comprising more than 5000 commits from 60 authors around the world, Spyder 4 finally saw the light in December 5th 2019! I decided to wait until now to make a formal announcement because quickly after the release we found several critical performance issues and some regressions with respect to Spyder 3, most of which are fixed now in version [4.1.0](https://github.com/spyder-ide/spyder/releases), released on <...>.
```

Comment:
```suggestion
After more than three years in development and more than 5000 commits from 60 authors around the world, Spyder 4 finally saw the light on December 5, 2019! I decided to wait until now to make a formal announcement because shortly after the initial release, we found several critical performance issues and some regressions with respect to Spyder 3, most of which are fixed now in [version 4.1.0](https://github.com/spyder-ide/spyder/releases), released on <...>.
```

"Saw the light" isn't particularly idiomatic for this context, but I'm not exactly sure what to replace it with that would fit your vision so I just left it.